pipeline/metadata/beam_metadata.py

```diff
@@ -153,3 +170,37 @@ def merge_tagged_answers_with_rows(
       if tags.ip == untagged_answer.ip:
         merge_satellite_answers(untagged_answer, tags)
   return row
+
+
+def merge_blockpages_with_answers(
```

Comment:
I'd use `add_page_fetch_to_answers` or something like that. Let's move away from `blockpage` as it's not specific to blockpages.

Reply:
done